src/ng/sce.js

```diff
@@ -27,6 +27,13 @@ var SCE_CONTEXTS = {
 
 // Helper functions follow.
 
+var UNDERSCORE_LOWERCASE_REGEXP = /_([a-z])/g;
```

Comment:
Nothing. ;) This is a purely internal function, only used to iterate once through a few constant strings during the initialization.